pkgs/development/python-modules/pyprecice/default.nix

```diff
@@ -15,7 +15,7 @@ buildPythonPackage rec {
   propagatedBuildInputs = [ numpy mpi4py precice ];
 
   doCheck = false; # Disable Test because everything depends on open mpi which requires network.
-  pythonImportChecks = [ "precice" ];
+  pythonImportsCheck = [ "precice" ];
```

Comment:
```suggestion
```
so this is running the same tests as doCheck, I would suggest to drop it.

Reply:
Feel free to open a PR.

Reply:
https://github.com/NixOS/nixpkgs/pull/162560

Reply:
No, it isn't and this shouldn't be dropped. Otherwise we have 0 test coverage.